ui/js/controllers/perf/graphs.js

```diff
@@ -686,16 +686,6 @@ perf.controller('GraphsCtrl', [
             }, alertHttpError);
         }
 
-        function alertHttpError(error) {
```

Comment:
This function was only used in one place and seems unrelated to the changes in `alerts.js` (which is what this PR was about)?